src/components/AnimatedCheckMark/index.ts

```diff
@@ -1,7 +1,9 @@
+import CircularAnimatedCheckMark from './CircularAnimatedCheckMark';
 import FadedAnimatedCheckMark from './FadedAnimatedCheckMark';
 
 const AnimatedCheckMark = Object.assign(FadedAnimatedCheckMark, {
   Faded: FadedAnimatedCheckMark,
```

Comment:
remove the default one from the list

Reply:
It will be changed in Variant 3 Pull Request...